package exceptions

import "errors"

var ErrUserAlreadyExists = errors.New("usuário já cadastrado")
